tools/CoqMakefile.in

```diff
@@ -78,6 +78,11 @@ else
 STDTIME?=command time -f $(TIMEFMT)
 endif
 
+ifneq (,$(COQBIN))
+# add an ending /, if it's already there cleanup
+COQBIN:=$(subst //,/,$(COQBIN)/)
```

Comment:
what about `$(realpath` or `$(abspath`?

Reply:
I want to change as little as possible from what the user passed in. I don't want to have to think of whether abspath would change semantics.

Reply:
I disagree, the meaning of a path is its absolute version.

If I pass `../../` then this is relative to `$PWD` and if the makefile does not resolve it ASAP then using it in any snippet that does `cd` is wrong, the user may not even know the makefile changes directory, he can't possibly rely on a relative path being interpreted in a base other than `$PWD`.

Reply:
Note that extra slashes don't change the meaning of a path. It may trick the makefile, since path matching is not up to these differences. But I don't think we match COQBIN, so either you don't normalize it, or you normalize it with a function that was designed to do so.